### ISSUE TITLE
Images need alt text for screenreader users

### ISSUE THREAD

**Issue by sarenji** [BloombergMedia/whatiscode#29]
* http://webaim.org/techniques/alttext/
* http://accessibility.psu.edu/images/

**Comment by tophtucker** [comment]
Good call. As @wawjr3d [put it](https://twitter.com/wawjr3d/status/609359572248231936), "the required attributes of an img tag are src AND alt". Will do.
